content/api/commands/clear.md

```diff
@@ -52,8 +52,9 @@ Pass in an options object to change the default behavior of `.clear()`.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`.clear()` yields the same subject it was given from the previous
-command.</li></List>
+- `.clear()` yields the same subject it was given.
```

Comment:
why drop `from the previous command.` from all of the API docs?
```suggestion
- `.clear()` yields the same subject it was given from the previous command.
```

Reply:
"From the previous command, query or assertion" is pretty verbose, and I don't think it actually adds any clarity. Assertions usually don't change the subject, but they can! So even before queries, this wasn't quite accurate.